File: src/main/java/com/nuutrai/reactor/entity/lang/VentEntity.java
package com.nuutrai.reactor.entity.lang;

import com.nuutrai.reactor.entity.Sellable;
import com.nuutrai.reactor.util.MultiTypeMap;
import org.bukkit.Material;

import java.util.ArrayList;

public abstract class VentEntity extends Sellable {
    public VentEntity(String id, Material block) {
        super(id, block);
    }

    @Override
    public void tick() {
        tick((Sellable[]) new ArrayList<Sellable>().toArray(), new MultiTypeMap());
    }
}

File: src/main/kotlin/com/nuutrai/reactor/entity/lang/VentEntity.kt
package com.nuutrai.reactor.entity.lang

import com.nuutrai.reactor.entity.Sellable
import com.nuutrai.reactor.util.MultiTypeMap
import org.bukkit.Material

abstract class VentEntity(id: String, block: Material?) : Sellable(id, block) {
	override fun tick() {
		tick(arrayOf(), MultiTypeMap())
	}
}
